samples/good-arc.rb

```diff
@@ -26,8 +26,8 @@
       10.times do |j|
         strokewidth sw * j
         size = (basesize + step * j) / 3
-        top = (self.height - size) / 2
-        left = (self.width - size) / 2
+        top = (height - size) / 2
+        left = (width - size) / 2
```

Comment:
Does this sample still run properly? I'm generally in favor of excluding self where possible, but I remember some samples hit problems because of our fussing with self inside the blocks.